sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/MigrationGuide.md

```diff
@@ -9,9 +9,16 @@ Familiarity with the `Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker` libr
 - [Migration benefits](#migration-benefits)
 - [General changes](#general-changes)
   - [Package and namespaces](#package-and-namespaces)
-  - [Authenticating](#authenticating)
-  - [Querying a question](#querying-a-question)
-  - [Chatting](#chatting)
+  - [Runtime Client](#runtime-client)
+    - [Authenticating runtime client](#authenticating-runtime-client)
+    - [Querying a question](#querying-a-question)
+    - [Chatting](#chatting)
+  - [Authoring Client](#authoring-client)
+    - [Authenticating authoring client](#authenticating-authoring-client)
+    - [Creating knowledge base](#creating-knowledge-base)
+    - [Updating knowledge base](#updating-knowledge-base)
+    - [Exporting knowledge base](#exporting-knowledge-base)
+    - [Deleting knowledge base](#deleting-knowledge-base)
```

Comment:
Be sure to check that all these exist / navigate to the bookmark. Seems like they should be correct but, for example, I think the automatic stubber doesn't do anything for words like "a" or "the", so line 14 may not navigate, for example.

Reply:
Just checked them all, they are working (including line 14).